src/wmtk/SimplicialComplex.cpp

```diff
@@ -201,6 +202,20 @@ bool SimplicialComplex::link_cond_bd_2d(const Mesh& m, Tuple t)
     return true;
 }
 
+bool SimplicialComplex::link_cond_bd_1d(const Mesh& m, Tuple t)
+{
+    assert(m.top_cell_dimension() == 1);
```

Comment:
I marked that conversation as "unresolved". At least I don't see how it was resolved.

Reply:
I have added new files to the simplex directory. The new link condition is defined within the simplex_collection class. To avoid confusion, I have use link_condition and link_condition_for_closed_trimesh instead.